src/main/java/core/basesyntax/MyHashMap.java

```diff
@@ -1,19 +1,112 @@
 package core.basesyntax;
 
+import java.util.Objects;
+
+@SuppressWarnings("unchecked")
 public class MyHashMap<K, V> implements MyMap<K, V> {
+    private static final int DEFAULT_INITIAL_CAPACITY = 1 << 4;
+    private static final float DEFAULT_LOAD_FACTOR = 0.75f;
+    private Node<K, V>[] hashTable;
+    private int threshold = DEFAULT_INITIAL_CAPACITY * (int) DEFAULT_LOAD_FACTOR;
```

Comment:
what will be `0.75` in `int`?
```suggestion
    private int threshold = (int) DEFAULT_INITIAL_CAPACITY * DEFAULT_LOAD_FACTOR;
```

Reply:
Because
Error: Incompatible types. Found: 'float', required: 'int'
Warning: Casting 'DEFAULT_INITIAL_CAPACITY' to 'int' is redundant